File: app/src/main/java/com/apache/fastandroid/demo/designmode/proxy/dynamic/RealSubject.java
package com.apache.fastandroid.demo.designmode.proxy.dynamic;

import com.tesla.framework.common.util.log.FastLog;

/**
 * Created by Jerry on 2019/2/2.
 */
public class RealSubject implements ISubject {
    @Override
    public void request() {
        FastLog.d("","RealSubject request");
    }
}

File: app/src/main/java/com/apache/fastandroid/demo/designmode/proxy/dynamic/RealSubject.kt
package com.apache.fastandroid.demo.designmode.proxy.dynamic

import com.tesla.framework.component.logger.Logger

/**
 * Created by Jerry on 2019/2/2.
 */
class RealSubject : ISubject {
    override fun request() {
        Logger.d("RealSubject:Handle request")
    }
}